.circleci/config.yml

```diff
@@ -37,25 +37,23 @@ persist_to_workspace_step: &persist_to_workspace_step
 jobs:
   black:
     docker:
-      - *test_runner
+      - image: circleci/python:3.8
     resource_class: *resource_class
     steps:
       - checkout
-      - *restore_cache_step
+      - run: sudo pip install tox
```

Comment:
really? `sudo` is needed here?

Reply:
It's not, actually, you're right. Removed.